src/ui/public/timepicker/timepicker.js

```diff
@@ -24,7 +24,9 @@ module.directive('kbnTimepicker', function (quickRanges, timeUnits, refreshInter
       to: '=',
       mode: '=',
       interval: '=',
-      activeTab: '='
+      activeTab: '=',
+      onFilterSelect: '=',
```

Comment:
@cjcenizal I thought about this too, but then I couldn't actually think of any real disadvantages of the way Lukas is doing it here which is much simpler to understand. Do you know what gotchas there might be with the current implementation?